src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    public static void readFile(ArrayList<duke.Task> tasks) {
+        try {
+            File file = new File("src/main/java/duke.txt");
```

Comment:
This could potentially be causing the error you had during the tutorial - perhaps use a relative path to create the text file aka use something like "data/duke.txt". If the file doesn't exist, catch the `FileNotFoundException` and create a new file and directory. 

https://nus-cs2113-ay2021s2.github.io/website/schedule/week6/topics.html#c-to-java-miscellaneous-topics-file-access